Library/Formula/wireshark.rb

```diff
@@ -3,13 +3,9 @@ class Wireshark < Formula
   homepage "https://www.wireshark.org"
 
   stable do
-    url "https://www.wireshark.org/download/src/all-versions/wireshark-1.12.8.tar.bz2"
-    mirror "https://1.eu.dl.wireshark.org/src/wireshark-1.12.8.tar.bz2"
-    sha256 "357e0a4e49525d80cdc740bb16539fcdb526ad38cc2ed6cabedafc9bdee5c7e7"
-
-    # Removes SDK checks that prevent the build from working on CLT-only systems
-    # Reported upstream: https://bugs.wireshark.org/bugzilla/show_bug.cgi?id=9290
-    patch :DATA
+    url "https://www.wireshark.org/download/src/all-versions/wireshark-2.0.0.tar.bz2"
```

Comment:
I might have a lead.

On my OS X 10.9.5 machine with Xcode 6.2 and CLT installed...

```
Xcode: 6.2 => /Applications/Xcode-6.2.app/Contents/Developer
CLT: 6.2.0.0.1.1424975374
```

...that `PCAP_TSTAMP_PRECISION_NANO` is only present in the Xcode 6.2 OS X 10.10 SDK's `pcap.h`, not in the OS X 10.9 SDK `pcap.h`, or in the CLT's `pcap.h`.  [This says](http://upstream.rosalinux.ru/compat_reports/libpcap/1.4.0_to_1.5.1/abi_compat_report.html) that `PCAP_TSTAMP_PRECISION_NANO` was introduced between libpcap 1.4.0 and 1.5.1. OS X 10.10 comes with libpcap [1.5.3](http://opensource.apple.com/release/os-x-1010/).

From the `brew config` part of the failed build:

```
Xcode: 6.2
CLT: 6.2.0.0.1.1424975374
```

I *think* this means that the CI server has that same Xcode 6.2 installed, but not "selected" with `xcode-select -s`. So, I think that means it's still building against the installed CLT libpcap headers, which have the older 1.3.0 `libpcap` without `PCAP_TSTAMP_PRECISION_NANO`.

I wasn't able to reproduce this on my own 10.9 machine, since I can't figure out how to get back to the pre-`xcode-select -s` state. I tried doing `sudo xcode-select -s /Library/Developer/CommandLineTools`, but I still ended up with `brew config` having a `=>` in the Xcode line, and the `brew install -s wireshark` still succeeded.

```
Xcode: 6.2 => /Applications/Xcode-6.2.app/Contents/Developer
CLT: 6.2.0.0.1.1424975374
```

Soooo, if this is the case, I think the builds which are succeeding when pointed at Xcode 6.2 on 10.9 machines which do not have `dupes/libpcap` installed might actually be broken even though the build succeeds, because they're compiled against the 1.5.3 libpcap headers, but linked against the 1.3.0 libpcap libraries, and might fail or perform incorrectly at runtime?

I thought we were building against the same SDKs as the OS version we're building on, in the case where it's available?

Anyway, the CMake scripts in Wireshark, in particular [`cmake/modules/FindPCAP.cmake`] seem like they [should be detecting whether it's present in this version of libpcap](https://github.com/wireshark/wireshark/blob/wireshark-2.0.0/cmake/modules/FindPCAP.cmake#L91) and not attempting to use it. And when I build it on my 10.9 machine, it seems to see it as absent.

```
checking for pcap_set_tstamp_precision... no
```